contracts/JBMultiTerminal.sol

```diff
@@ -470,7 +468,7 @@ contract JBMultiTerminal is JBOperatable, Ownable, ERC2771Context, IJBMultiTermi
             _heldFee = _heldFees[_i];
 
             // Get the fee amount.
-            _amount = (_heldFee.fee == 0 ? 0 : JBFees.feeIn(_heldFee.amount, _heldFee.fee));
+            _amount = (JBFees.feeIn(_heldFee.amount, FEE));
```

Comment:
JBFees.feeIn might be useful elsewhere (but not currently used elsewhere, correct me if I'm wrong), and it's all being used with the same FEE amount here.

We could save a bit of space by hardcoding it into the function, but might not be worth the hassle.

Reply:
i think we're good on space as is! nice keeping the fee calculations in their own zone